src/API/Loggy.Api/Startup.cs

```diff
@@ -33,8 +33,9 @@ public void ConfigureServices(IServiceCollection services)
 			services.AddSingleton<RootQuery>();
 			services.AddSingleton<MySubQuery>();
 			services.AddSingleton<LogEntriesQuery>();
+            services.AddSingleton<MySubObjectGraphType>();
```

Comment:
Ah nice one JP - do you want to post it here too? [link](https://stackoverflow.com/questions/53773837/why-is-graphql-dotnet-returning-expected-non-null-value-error-for-this-schema)

Reply:
Cool thanks Ryan :D